### ISSUE TITLE
What is the ori_shading.exr?

### ISSUE THREAD

**Issue by changbozhu** [zhhoper/DPR#14]
Thank you for your sharing!

Well, I have a question about the DPR_dataset. There is a file named ori_shading.exr, and I cannot find it in RI_render_DPR's result. So how to make the ori_shading.exr, is this the same file with ori_shading.npy?

**Comment by zhhoper** [comment]
Yes, ori_shading.npy is the same with ori_shading.exr. *.exr requires openexr to load the file which may not be easy to install so I save it as *.npy
